### ISSUE TITLE
Responding to Pub Board and SAC Reviewers' Feedback 

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTScienceCollaborations/ObservingStrategy#621]
tl;dr: We've got a lot of high quality feedback on the LSST Observing Strategy White Paper coming in from the Science Advisory Committee, and it's up to the Chapter Editors to respond (delegating as needed). Let's get everything squared away by May 26th and be on the arxiv in June!

Hi @LSSTScienceCollaborations/observing-strategy-white-paper-editors !

As you'll have noticed, the comments from the Science Advisory Committee reviewers is flowing in, via their coordinator, @michaelstrauss. Thanks very much for getting us this valuable feedback, Michael! 

As I look through their comments in the issue threads, I am assigning those issues to the relevant chapter editors to address their comments and provide a response. (You may want to issue your section teams with responding to specific things.) I think you should treat these reviews as you would a referee report: please make the recommended changes that you (and your section teams)  agree with, and use the issue threads to provide a written response to any requests that you cannot or choose not to satisfy. This is important: the reviewers spent some of their time on us, so we must thank them by responding well. I think you'll find their feedback will improve your chapters, enhancing your reputations as a result! :-)

We need a deadline to get the reviews responded to and the paper finished. Let's go for **Friday May 26th!** That gives us more than a month to resolve all of our issues, which seems pretty doable to me. Talking of issues, here are some links that you'll find useful as you return to the white paper:

* [All issues marked "LSST Review"](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues?q=is%3Aissue+is%3Aopen+label%3A%22LSST+review%22), to see what your chapter's feedback is

* [All issues in the "Version 1.0 Submission to ArXiV" milestone](https://github.com/LSSTScienceCollaborations/ObservingStrategy/milestone/14) 

While you're at it, please do look through your older issues as well: we still have issues attached to the ["Tucson 2016 Project and Community Meeting" milestone](https://github.com/LSSTScienceCollaborations/ObservingStrategy/milestone/16) that can now either be closed (please close them!), moved to the Version 1.0 milestone (please move them!), or punted into the [next version](https://github.com/LSSTScienceCollaborations/ObservingStrategy/milestone/20) (please move them!). If you do move an issue from one milestone to another, please drop a comment in the box as you do, so we all know what's going on. 

I hope this all sounds OK - comments/queries/suggestions welcome, as always! :-)

Thanks very much for putting in the work needed to get this onto the arxiv!  I'm going to re-order the author list to put you editors in a block at the front - seems only fair given the amount of work you've all put in over the last couple of years!

Cheers

Phil

**Comment by drphilmarshall** [comment]
Hi all!

We're now a bit over a week away from our May 26th deadline for addressing all our SAC and Project reviewers' comments. Lots of pull requests have been coming in, thank you! Please keep them coming so we can get the white paper onto the arxiv in June as planned. 

To find all the issues from the v1.0 submission milestone that are assigned to you, start from [this link](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Version+1.0+Submission+to+ArXiV%22) and then filter further by assignee. You can also browse all of the feedback we got from our reviewers (and that has been issued) [here](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues?utf8=%E2%9C%93&q=is%3Aopen%20is%3Aissue%20milestone%3A%22Version%201.0%20Submission%20to%20ArXiV%22%20label%3A%22LSST%20review%22%20).

Cheers!

Phil

**Comment by StephenRidgway** [comment]
OK, tomorrow.  I have not edited for some time.  Is the current doc in the same place on github and I use the same process for editing as always?

Steve

**Comment by drphilmarshall** [comment]
Yes please! Thanks very much for making time for this, Steve. Do please
make sure you do a `git pull` before you start editing your local files.

**Comment by drphilmarshall** [comment]
HI all! Today is our deadline day for responding to our LSST reviews :-) Thanks very much for all your pull requests so far - I'll be working on merging these as they come in, as well as working on my own issues... 

To see what's left for you to do, please do filter by assignee
 
* [all issues marked "LSST Review"](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues?q=is%3Aissue+is%3Aopen+label%3A%22LSST+review%22), to see what your chapter's feedback is

and/or

* [all issues in the "Version 1.0 Submission to ArXiV" milestone](https://github.com/LSSTScienceCollaborations/ObservingStrategy/milestone/14) 

to see what's assigned to you.

While you're at it, please do look through your older issues as well: we still have issues attached to the ["Tucson 2016 Project and Community Meeting" milestone](https://github.com/LSSTScienceCollaborations/ObservingStrategy/milestone/16) that can now either be closed (please close them!), moved to the Version 1.0 milestone (please move them!), or punted into the [next version](https://github.com/LSSTScienceCollaborations/ObservingStrategy/milestone/20) (please move them!). If you do move an issue from one milestone to another, please drop a comment in the box as you do, so we all know what's going on. 

Thanks again for all your work on this paper - with all your attention to detail it's coming together very nicely indeed! :-)

**Comment by drphilmarshall** [comment]
Apologies for the slew of issue notifications that's about to start! I have finally found some time right before the Tucson meeting to close out the final review issues and get the v1.0 white paper submitted. I think I can just do it, and Beth and Zeljko have signed off, so here we go - I'll close this one out when the review issues are all closed, and then find somewhere else to post the arxiv link.
